### PR TITLE
Eip 186

### PR DESCRIPTION
**Pull request by bsieber-mozilla**

### CODE REVIEW

**Comment by bsieber-mozilla** [comment]
- [ ] pre-commit run on files
- [ ] Behave + deps example for future testing harness
- [ ] Tox wrapper around testing strategy for  consistent testing harness on ci and local 
- [ ] Converting to pyproject.toml for setup, using this conf for tool conf as well
- [ ] pipfile.lock removed from git

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+version = "0.1.0"
+description = "Template python repo with our proven approaches."
+authors = [
+    "Adam Frank <afrank@mozilla.com",
+    "Bryan Sieber <bsieber@mozilla.com>",
+    "Chris Valaas <cvalaas@mozilla.com",
+    "Jorge Spiropulo <jspiropulo@mozilla.com"
+]
+readme = "README.md"
+python = "^3.7"
```
Comment (mozafrank) [inline pyproject.toml:12]: @bsieber-mozilla I'm already using python 3.8 locally, can we specify ">=3.4" unless anyone disagrees?

Comment (bsieber-mozilla) [reply pyproject.toml:12]: Oops, I thought this was a >=3.7! Want >=3.7 or 3.4?

Comment (mozafrank) [reply pyproject.toml:12]: I've been using >=3.4 but I'd also be totally fine with >=3.7 at this point.

Comment (bsieber-mozilla) [reply pyproject.toml]: Swapped it to >=!

**Pull request merged**